dataprofiler/data_readers/structured_mixins.py

```diff
@@ -72,12 +72,12 @@ def _get_data_as_df(self, data: pd.DataFrame) -> pd.DataFrame:
     def _get_data_as_records(self, data: Any) -> List[str]:
         """Return data records."""
         records_per_line = min(len(data), self.SAMPLES_PER_LINE_DEFAULT)
-        data = [
+        data_ = [
             str(
                 "\n".join(data[i * records_per_line : (i + 1) * records_per_line])
                 .encode("UTF-8")
                 .decode()
             )
             for i in range((len(data) + records_per_line - 1) // records_per_line)
         ]
-        return data
+        return data_
```

Comment:
is `data` conflicting with a python keyword here?

Reply:
The parameter name is also `data` so I had to change the name because we're returning a different type.

Reply:
+1 CC @Sanketh7

Reply:
To get around the changing types, I decided to cast `data` before returning it.